internal/handlers/sqlite/msg_dropdatabase.go

```diff
@@ -59,6 +59,8 @@ func (h *Handler) MsgDropDatabase(ctx context.Context, msg *wire.OpMsg) (*wire.O
 	switch {
 	case err == nil:
 		res.Set("dropped", dbName)
+	case backends.ErrorCodeIs(err, backends.ErrorCodeDatabaseNameIsInvalid):
+		// nothing?
```

Comment:
I guess we can decide based on what MongoDB does for dropping invalid names, right?

Reply:
Yeah, a test would be needed there, but that's not super important